src/components/Venues/VenueCard.tsx

```
@@ -62,33 +62,40 @@ const VenueCard = ({
             className={styles.novenueimage}
           />
           <Card.Body className="pb-0">
             <Card.Title className="d-flex justify-content-between">
               {/* Venue name with truncation if too long */}
-              <div className={styles.title}>
+              <div
+                className={styles.title}
+                data-testid={`venue-name${index + 1}`}
+              >
                 {venueItem.name.length > 25
                   ? venueItem.name.slice(0, 25) + '...'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)